guacml/models/base_model.py

```diff
@@ -18,8 +18,15 @@ def train(self, x, y, **kwargs):
     def predict(self, x):
         raise NotImplementedError()
 
+    def feature_importances(self, x):
+        raise NotImplementedError()
+
     @staticmethod
     def to_int(value):
         if value is None:
             return None
-        return int(value)
\ No newline at end of file
+        return int(value)
+
+    @staticmethod
+    def pos_int(value):
+        return max(int(value), 0)
```

Comment:
Did you know about the [convention](https://robots.thoughtbot.com/no-newline-at-end-of-file) that every text file (incl source code) should end with a newline? If you change your editor to automatically add them this weird git errors 'no newline at end of file' will go away.